ops/op2/dispatch_fast.rs

```diff
@@ -713,6 +713,17 @@ fn map_v8_fastcall_arg_to_arg(
       )?;
       quote!(let #arg_ident = #fast_api_callback_options.wasm_memory as _; #convert; let #arg_ident = Some(#arg_ident);)
     }
+    Arg::Special(Special::CppGcResource(ty)) => {
+      *needs_fast_api_callback_options = true;
+      quote! {
+        let #arg_ident = #arg_ident.try_into().unwrap();
+        let Some(#arg_ident) = deno_core::cppgc::unwrap_cppgc_object::<#ty>(#arg_ident) else {
```

Comment:
`unwrap_cppgc_object` should be `try_unwrap_cppgc_object`

I think we may also want to change that method to take either `&v8::Local<...>` or `&v8::Object` so that the caller must hold the reference to the v8 object while the Rust object is alive.

Reply:
Renamed to `try_unwrap`. 

Yeah I think we want to add lifetime restrictions but shouldn't be a problem for sync ops.